third_party/llvm/macos_build_fix_and_amdgpu_gfx90a_fix.patch

```diff
@@ -0,0 +1,335 @@
+From 1ebfdd7ba05aa861df0397caf4c910c4d6a2690f Mon Sep 17 00:00:00 2001
+From: Rahul Joshi <jurahul@google.com>
+Date: Wed, 29 Sep 2021 11:13:44 -0700
+Subject: [PATCH] Revert "[Bazel] Add support for targeting macOS arm64"
+
+This reverts commit 33e1713a00a5291e5de658d0eb0aebdbf1d3aa03.
+---
+ utils/bazel/llvm-project-overlay/llvm/config.bzl | 3 +--
+ 1 file changed, 1 insertion(+), 2 deletions(-)
+
+diff --git a/utils/bazel/llvm-project-overlay/llvm/config.bzl b/utils/bazel/llvm-project-overlay/llvm/config.bzl
```

Comment:
Why merge these? Can we not have more than one patch?

Reply:
i don't think so.